plugins/woocommerce-blocks/assets/js/icons/library/customer-account-style-line.tsx

```diff
@@ -4,11 +4,24 @@
 import { SVG } from '@wordpress/primitives';
 
 const customerAccountStyleLine = (
-	<SVG xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
+	<SVG
+		width="12"
+		height="13"
+		viewBox="0 0 12 13"
+		fill="none"
+		xmlns="http://www.w3.org/2000/svg"
+	>
+		<circle
+			cx="6"
+			cy="3"
+			r="2.25"
+			stroke="currentColor"
+			stroke-width="1.5"
+		/>
 		<path
-			fillRule="evenodd"
-			clipRule="evenodd"
-			d="M10.1919 3.69231C10.1919 4.90309 9.21039 5.88462 7.99961 5.88462C6.78884 5.88462 5.80731 4.90309 5.80731 3.69231C5.80731 2.48153 6.78884 1.5 7.99961 1.5C9.21039 1.5 10.1919 2.48153 10.1919 3.69231ZM11.6919 3.69231C11.6919 5.73151 10.0388 7.38462 7.99961 7.38462C5.96041 7.38462 4.30731 5.73151 4.30731 3.69231C4.30731 1.6531 5.96041 0 7.99961 0C10.0388 0 11.6919 1.6531 11.6919 3.69231ZM11.6919 10.4615H4.30731C3.2877 10.4615 2.46115 11.2881 2.46115 12.3077V16H0.614998V12.3077C0.614998 10.2685 2.2681 8.61539 4.30731 8.61539H11.6919C13.7311 8.61539 15.3842 10.2685 15.3842 12.3077V16H13.5381V12.3077C13.5381 11.2881 12.7115 10.4615 11.6919 10.4615Z"
+			fill-rule="evenodd"
+			clip-rule="evenodd"
```

Comment:
Should these attributes keep using camel case?

Reply:
I _think_ it's ok to keep these hyphenated since the `SVG` component is just a wrapper that returns a native `svg` element with the required x-browser accessibility attributes. The native `viewBox` attr is camelcase (for whatever reason) while these other items are hyphenated, as I understand it.

That being said, I do think we should import and use the `Path` and `Circle` exports from the `SVG` package as well ([reference](https://github.com/WordPress/gutenberg/blob/trunk/packages/primitives/src/svg/index.js)).